deploy/crds/mongodb.com_v1_mongodb_scram_cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: mongodb.com/v1
+kind: MongoDB
+metadata:
+  name: example-scram-mongodb
+spec:
+  members: 3
+  type: ReplicaSet
+  version: "4.2.6"
+  security:
+    authentication:
+      enabled: true
+      modes: ["SCRAM"]
```

Comment:
CRD changes will be included in E2E test, currently this file isn't actually being used, right now it's just a sample as to what it will look like